x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -112,6 +112,27 @@ export interface ResolverChildNode extends ResolverLifecycleNode {
   nextChild?: string | null;
 }
 
+/**
```

Comment:
For now I'm duplicating all the types that are used between the frontend and backend. The backend uses the safe versions and the frontend will use the unsafe ones for now.